### PR TITLE
Add unit tests for Label

### PR DESCRIPTION
**Pull request by Olina-Zhang**
related https://github.com/dotnet/winforms/issues/10453

## Proposed changes
Add unit tests for Label to test its BorderStyle, FlatStyle and TextAlign

## Customer Impact

- None

## Regression? 

- No

## Risk

- Minimal

## Test methodology 

- Unit tests

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/10787)

### CODE REVIEW

**Review by Tanya-Solyanik — COMMENTED** [review]
Looks good, I added only minor comments. Please see https://github.com/dotnet/winforms/blob/main/docs/coding-style.md

**Review by SimonZhao888 — APPROVED** [review]

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs [hunk 1]
```diff
@@ -299,6 +299,128 @@ public void Label_AutoSize_BehavesExpected(bool autoSize, bool expected)
...
+    }
+
+    [WinFormsTheory]
+    [InlineData(FlatStyle.System)]
+    [InlineData(FlatStyle.Popup)]
+    [InlineData(FlatStyle.Standard)]
+    [InlineData(FlatStyle.Flat)]
+    public void Label_FlatStyle_Set_GetReturnsExpected(FlatStyle style)
+    {
+        using Label label = new Label();
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs:355]:
```suggestion
        using Label label = new();
```

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs [hunk 2]
```diff
@@ -299,6 +299,128 @@ public void Label_AutoSize_BehavesExpected(bool autoSize, bool expected)
...
+    [InlineData(ContentAlignment.TopRight)]
+    [InlineData(ContentAlignment.MiddleLeft)]
+    [InlineData(ContentAlignment.MiddleCenter)]
+    [InlineData(ContentAlignment.MiddleRight)]
+    [InlineData(ContentAlignment.BottomLeft)]
+    [InlineData(ContentAlignment.BottomCenter)]
+    [InlineData(ContentAlignment.BottomRight)]
+    public void Label_TextAlign_Set_GetReturnsExpected(ContentAlignment alignment)
+    {
+        using Label label = new Label();
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs:391]:
```suggestion
        using Label label = new();
```

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs [hunk 3]
```diff
@@ -299,6 +299,128 @@ public void Label_AutoSize_BehavesExpected(bool autoSize, bool expected)
...
+        label.TextAlign = alignment;
+
+        Assert.Equal(alignment, label.TextAlign);
+        Assert.True(label.OwnerDraw);
+    }
+
+    [WinFormsFact]
+    public void Label_TextAlign_SetSameValue_DoesNotInvalidate()
+    {
+        using Label label = new Label();
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs:401]:
```suggestion
        using Label label = new();
```

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs [hunk 4]
```diff
@@ -299,6 +299,128 @@ public void Label_AutoSize_BehavesExpected(bool autoSize, bool expected)
...
+        Assert.True(label.IsHandleCreated);
+
+        label.TextAlign = ContentAlignment.TopLeft;
+        Assert.True(label.IsHandleCreated);
+    }
+
+    [WinFormsFact]
+    public void Label_TextAlign_SetDifferentValue_Invalidate()
+    {
+        using Label label = new Label();
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs:414]:
```suggestion
        using Label label = new();
```

**Comment by Tanya-Solyanik** [comment]
@SimonZhao888  - it looks like this PR did not generate a code coverage report, could you please ensure that even changes to unit test projects generate code coverage reports?

**Comment by SimonZhao888** [comment]
> @SimonZhao888 - it looks like this PR did not generate a code coverage report, could you please ensure that even changes to unit test projects generate code coverage reports?

OK, got it.

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs (hunk 1 above)
Comment (Olina-Zhang) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LabelTests.cs]: Thanks @Tanya-Solyanik, sorry for that.

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you!

**Pull request merged**